packages/accordion/src/Accordion.ts

```diff
@@ -41,6 +41,12 @@ export class Accordion extends SpectrumElement {
     @property({ type: Boolean, reflect: true, attribute: 'allow-multiple' })
     public allowMultiple = false;
 
+    @property({ type: String, reflect: true })
+    public density?: 'compact' | 'regular' | 'spacious' = 'regular';
+
+    @property({ type: String, reflect: true })
+    public size?: 's' | 'm' | 'l' | 'xl' = 'm';
```

Comment:
We no longer size our components with an attribute like this. Rather, we use the `SizedMixin` helper as an extension (see a usage of this [here](https://github.com/adobe/spectrum-web-components/blob/6d5d6b3c31d92a058202a242a283125d25131f81/packages/tags/src/Tag.ts#L35)). This should simplify the implementation for you as you won't need the `setAccordionProperties()` method anymore.